cmd/exporters/prometheus/prometheus.go

```diff
@@ -505,7 +521,7 @@ func (p *Prometheus) render(data *matrix.Matrix) ([][]byte, error) {
 
 			if p.addMetaTags && !tagged.Has(prefix+"_"+metric.GetName()) {
 				tagged.Add(prefix + "_" + metric.GetName())
-				rendered = append(rendered, []byte("# HELP "+prefix+"_"+metric.GetName()+" Metric for "+data.Object))
+				rendered = append(rendered, []byte("# HELP "+prefix+"_"+metric.GetName()+description+" OBJECT: "+data.Object))
```

Comment:
I think histograms need more work on the zapiperf and restpef side. Thinking that we should probably include the labels as part of the text